oak/server/rust/oak_abi/build.rs

```diff
@@ -14,12 +14,15 @@
 // limitations under the License.
 //
 
+use std::path;
+
 fn main() {
-    oak_utils::run_protoc_rust(protoc_rust::Args {
-        out_dir: "src/proto",
-        input: &["../../../../oak/proto/oak_api.proto"],
-        includes: &["../../../../oak/proto"],
-        customize: protoc_rust::Customize::default(),
-    })
-    .expect("protoc");
+    let proto_dir = path::Path::new("../../../../oak/proto/");
+    let oak_api_path = proto_dir.join("oak_api.proto");
+
+    // Tell Cargo that if the given file changes, to rerun this build script.
+    // https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed
+    println!("cargo:rerun-if-changed={:?}", oak_api_path);
```

Comment:
I think this may be:

```suggestion
    println!("cargo:rerun-if-changed={}", oak_api_path.to_str().unwrap());
```

Reply:
Done